docs/tutorials/services/data-mlmodel-tutorial.md

```diff
@@ -0,0 +1,322 @@
+   By default, the data management service captures data to the <file>~/.viam/capture</file> directory, and syncs captured data files to the Viam app every 6 seconds (`0.1` minutes in the configuration).
+   Leave the default settings as they are, and click **Save Config** at the bottom of the screen to save your changes.
+
+   {{< imgproc src="/tutorials/data-management/data-management-conf.png" alt="The data management service configuration pane with default settings shown for both capturing and syncing" resize="600x" >}}
+
+For more information, see [Add the data management service](/services/data/configure-data-capture/#add-the-data-management-service).
+
+### Configure data capture for a component
+
+Once you have added the data management service, you can configure data capture for specific components on your smart machine.
+For this tutorial, you will configure data capture for images from a [camera](/components/camera/) component, but other data types such as sensor data or SLAM map data from other types of [components](/components/) can be captured as well.
+
+To enable image data capture for a camera component:
+
+1. On your robot's **Config** page in the [Viam app](https://app.viam.com), navigate to the **Components** tab.
+1. If you have not already, add a [camera component](/components/camera/), such as a [webcam](/components/camera/webcam/), to your smart machine.
+   If you are using a Viam Rover, the camera is pre-configured, and named `cam`.
+
+1. In your camera component configuration pane, find the **Data capture configuration** section, and click the **Add method** button to enable data capture for this camera.
+
+   - Set the **Type** to `ReadImage` and the **Frequency** to `0.333`.
+     This will capture an image from the camera roughly once every 3 seconds.
+     You can adjust the capture frequency if you want the camera to capture more or less image data, but avoid configuring data capture to higher rates than your hardware can handle, as this could lead to performance degradation.
+
+   - Set the **Mime type** to `image/jpeg` to configure image data capture.
+
+     {{< imgproc src="/tutorials/data-management/camera-data-capture.png" alt="The camera component configuration pane with data capture configuration enabled using type ReadImage and a capture frequency of 0.333" resize="600x" >}}
+
+1. Click **Save Config** at the bottom of the window to save your changes.
+
+For more information see [Configure data capture](/services/data/configure-data-capture/#configure-data-capture-for-individual-components) and [Configure cloud sync](/services/data/configure-cloud-sync/).
+
+### View and filter captured data
+
+Now that you have configured data capture on your camera component, you can view the resulting data files in the Viam app, and can apply filters to limit the files returned by specific criteria, such as by robot or date range.
```

Comment:
```suggestion
Now that you have configured data capture on your camera component, you can view the resulting data files in the Viam app.
You also have the option to apply filters—whether by robot, date range, or other criteria—to limit the returned files.
```
It may just be my specific brain, so please feel free to disregard. I paused at "to limit the files returned by specific criteria" because there are a few ways to interpret that, so I thought this might help a bit. Definitely a subjective experience!

Reply:
Much better, thank you!